### PR TITLE
Instructions for installing locally

### PR DESCRIPTION
**Pull request by BLaurenB**
Just looking for comments to make sure this is on the right track. This only includes basic instructions for users who just want to use the vetter locally.  This is not intended for users who want to build in order to contribute. That's the next step.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,8 +158,68 @@ It includes:
...
 completed and submitted the [F5® Contributor License Agreement](https://github.com/aspenmesh/cla/raw/master/f5-cla.pdf)
 to [cla@aspenmesh.io](mailto:cla@aspenmesh.io) prior to their code submission
 being included in this project. Please include your github username in the CLA email.
+
+### Build Prerequisites
+To build Istio-Vet locally, you will need to install the following:
+
+* A [Go environment](https://golang.org/doc/install).
+* Install [dep](https://golang.github.io/dep/docs/installation.html)
+* Install [Protobuf](https://github.com/golang/protobuf)
```
Comment (nrjpoddar) [inline README.md]: Let's break this up in two sections for Mac and Linux users so that they can easily follow along.

Comment (nrjpoddar) [reply README.md]: Instructions for protoc: http://google.github.io/proto-lens/installing-protoc.html

File: README.md [hunk 2]
```diff
@@ -123,8 +158,68 @@ It includes:
...
+    https://github.com/google/protobuf/releases/download/<desired-version>/protoc-<desired-version>-linux-x86_64.zip \
+    && mkdir -p /usr/local \
+    && unzip protoc-<desired-version>-linux-x86_64.zip -d /usr/local
+    ```
+    You should be able to type `protoc` at the command line now and see its options.
+    Next, get protobuf:
+    ```bash
+    go get -u github.com/golang/protobuf/protoc-gen-go
+    ```
+* Install [Mercurial](https://www.mercurial-scm.org/) if not already installed. Mac users can use Brew. Linux users can run `apt-get install mercurial`
```
Comment (nrjpoddar) [inline README.md]: Verify if we need mercurial.

File: README.md [hunk 3]
```diff
@@ -123,8 +158,68 @@ It includes:
...
+* Make this directory. (Dependencies rely on this file structure)
+    ```bash
+    go/src/github.com/aspenmesh/
```
Comment (nrjpoddar) [inline README.md]: mkdir -p $GOPATH/src/github.com/aspenmesh; cd $GOPATH/src/github.com/aspenmesh

File: README.md [hunk 4]
```diff
@@ -123,8 +158,68 @@ It includes:
...
+    ```
+
+* Fork and clone this repo into your aspenmesh folder, then cd into istio-vet
```
Comment (nrjpoddar) [inline README.md]: `git clone <your-namespace>istio-vet`; cd `istio-vet`.

File: README.md [hunk 5]
```diff
@@ -36,14 +38,44 @@ Message: "WARNING: The pod myapp-xyz-1234 in namespace default is running with
...
+
+* From within your Go environment, navigate to the directory where you want Istio-Vet to be installed.
+* Make sure you have Go installed in your Path:
+    ```bash
+    export GOPATH=$HOME/gocode
+    export PATH=$PATH:$GOPATH/bin
+    ```
+* Run the go command to get Istio-Vet:
+    ```bash
+    go get github.com/aspenmesh/istio-vet/cmd/vet
```
Comment (nrjpoddar) [inline README.md]: Just add a link here which refer to the Contributing section for building istio-vet locally, then we can remove this.

File: README.md [hunk 6]
```diff
@@ -36,14 +38,44 @@ Message: "WARNING: The pod myapp-xyz-1234 in namespace default is running with
 sidecar proxy version 0.2.10 but your environment is running Istio
 version 0.2.12. Consider upgrading the sidecar proxy in the pod."
 ```
+<br>
+<br>
```
Comment (nrjpoddar) [inline README.md]: Remove one or both as you deem fit.

File: README.md (hunk 2 above)
Comment (BLaurenB) [reply README.md]: Our official Istio-Vet image doesn't have it, so I'm taking it out. :)

**Pull request merged**